<!DOCTYPE html>
<html lang="en-us">

  <head>
  

    <meta charset="UTF-8">
    <title>Deltaretrovirus-GLUE by giffordlabcvr: Paleovirus data</title>
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <link rel="stylesheet" type="text/css" href="stylesheets/normalize.css" media="screen">
    <link href='https://fonts.googleapis.com/css?family=Open+Sans:400,700' rel='stylesheet' type='text/css'>
    <link rel="stylesheet" type="text/css" href="stylesheets/stylesheet.css" media="screen">
    <link rel="stylesheet" type="text/css" href="stylesheets/github-light.css" media="screen">

  </head>

  <body>
    <section class="page-header">
    
      <h1 class="project-name">Deltaretrovirus-GLUE</h1>
      <h2 class="project-tagline">Resources for comparative genomic analysis of deltaretroviruses.</h2>

      <a href="../index.html" class="btn">Home</a>
      <a href="./viruses.html" class="btn">Viruses</a>
      <a href="https://github.com/giffordlabcvr/Deltaretrovirus-GLUE/zipball/master" class="btn">Download</a>
      <a href="https://github.com/giffordlabcvr/Deltaretrovirus-GLUE/tree/master" class="btn">View on GitHub</a>

    </section>

    <section class="main-content">

	   <h3>
		 <a id="Background" class="anchor" href="#Background" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Endogenous deltaretrovirus (ERV) data</strong>
	   </h3>

		 <p>

		   <b>Endogenous retroviruses (ERVs)</b> are retrovirus-derived DNA sequences that
		   are commonly found in vertebrate genomes.
		   ERV sequences are thought to have originated via ancient ‘germline incorporation’
		   events in which retroviruses infected
		   germline cells, generating retroviral proviruses that were subsequently
		   inherited as novel host alleles. 
		 
		 

		 </p>


         <p><img src="./images/paleovirus-header.jpg" alt="Species with endogenous deltaretrovirus" /></p>



<blockquote>
         <b>Species that have deltaretrovirus-derived ERVs in their genomes</b>, left to right:   
         <i>Rhinolophus bats</i>; <i>Anoura</i> bats; <i>Miniopterus</i> bats;
         the Indus River dolphin (<i>Platanista minor</i>).
</blockquote>
		 


		 <p>
	
		   ERVs comprise a unique source of information about the long-term
		   evolution of exogenous retroviruses. ERVs derived from deltaretroviruses are extremely scarce,
		   but a few have been identified. Information about these deltaretroviral ERVs
		   is collated in <b>Deltaretrovirus-GLUE</b>.

		 </p>

 
 	   <br>

	   <h3>
		 <a id="Virology" class="anchor" href="#sequenceData" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Relevance to molecular biological studies of deltaretroviruses</strong>
	   </h3>        
	   <hr>

	   <p>
          Endogenous viral sequences can inform our understanding of contemporary viruses
          in a wide variety of ways. Perhaps most importantly, ERVs allow calibration of
          the long-term evolutionary history of virus groups, which greatly influences 
          how we understand their biology.
 	   </p>

	   <p>
         Importantly, once time calibrations have been established, a far richer range of
         comparative genomic studies can be performed. By examining
         variation in the light of a known evolutionary history, these studies can provide 
         invaluable insights into the biological mechanisms through which viruses 
         replicate and spread.
	   </p>


 
 	   <br>
	   <h3>
		 <a id="EveProject" class="anchor" href="#EveProject" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>The ERV component of Deltaretrovirus-GLUE</strong>
	   </h3>
	   <hr>

	   
		<p>

          Currently, the distribution and diversity of deltaretrovirus-related sequences
          in animal genomes remains incompletely characterized.
          Progress in characterising these elements has been hampered by the challenges
          encountered attempting to analyse their fragmentary and degenerated sequences. 
          <b>Deltaretrovirus-GLUE</b> aims to address these issues.
		</p>

	   
		<p>
          We have incorporated into this project a set of principles for
          organising the deltaretrovirus 'fossil record', and a protocol 
          through which it can be accessed and collaboratively developed.
		</p>

        <p>
          This website provides background information about the ERV component of 
          <b>Deltaretrovirus-GLUE</b>, and direct links to specific data items. 

		</p>

 	   <br>
	   <h3>
		 <a id="Nomenclature" class="anchor" href="#Nomenclature" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Nomenclature for ERVs</strong>
	   </h3>
	   <hr>

		<p>
          We have applied a systematic approach to naming ERV, following a convention 
          developed for <a target="_blank" href="https://retrovirology.biomedcentral.com/articles/10.1186/s12977-018-0442-1">endogenous retroviruses</a>.
		  Each element was assigned a unique identifier (ID) constructed from a defined
		  set of components.
  		</p>


 
        <p><img src="./images/nomenclature-delta.jpg" alt="ERV Nomenclature" /></p>

 		


		<p>
          The first component is the classifier ‘ERV’ (endogenous retrovirus). 
		</p>

		<p>
          The second component is a composite of two distinct subcomponents separated by a period:
          
          (i)  the name of ERV group;
          (ii) a numeric ID that uniquely identifies the insertion.
          The numeric ID is an integer that identifies a unique insertion locus that arose as a
          consequence of an initial germline infection. Thus, orthologous copies in different
          species are given the same number. 
 		</p>
		
		<p>
          Where an ERV sequence is thought to have been duplicated within the germline
          following it's initial incorporation (e.g. via segmental duplication or transposition)
          we have appended an additional 'duplicate id' to the numeric ID, separated by a period.
          Please note that we have not yet resolved the orthologous relationships among
          sets of ERV sequences belonging to multicopy ERV lineages. We
          have therefore assigned unique duplicate IDs to each sequence within these lineages. 
		</p>
		
		<p>
          The third component of the ID defines the set of host species in which
          the ortholog occurs, or did occur prior to being deleted.
  		</p>

 
	   <br>
	   <h3>
		 <a id="ErvLocusData" class="anchor" href="#Raw_digs_data" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><b>Individual ERV loci: sequences and data</b>
	   </h3>
	   <hr>

		<p>
          These sequences were recovered via similarity searches performed using the
          <a target="_blank" href="https://giffordlabcvr.github.io/DIGS-tool/">database-integrated genome screening (DIGS) tool</a>
          in whole genome sequence (WGS) data obtained from the
          <a href="https://www.ncbi.nlm.nih.gov/genome/browse "><b>NCBI genomes</b></a> resource
          (2020-07-15).
		</p>


		<p>
           Information about the ERV sequences in tabular format can be found <a target="_blank" href="https://github.com/giffordlabcvr/Deltaretrovirus-GLUE/blob/master/tabular/erv/erv-locus-data.tsv"><strong>here</strong></a>.
  		</p>

		<p>
           Nucleotide level data in FASTA format (individual files) can be found <a target="_blank" href="https://github.com/giffordlabcvr/Deltaretrovirus-GLUE/tree/master/sources/fasta-digs-erv"><strong>here</strong></a>.
  		</p>
		
        
	   <br>
	   <h3>
		 <a id="schemaExtensions" class="anchor" href="#sequenceData" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Paleovirus-specific schema extensions in Deltaretrovirus-GLUE</strong>
	   </h3>
	   <hr>


		 <p>
		   The paleovirus component of Deltaretrovirus-GLUE extends GLUE's
		   <a target="_blank" href="http://glue-tools.cvr.gla.ac.uk/#/coreSchema"><b>core schema</b></a>
		   to allow the capture of ERV-specific data.
		   These schema extensions are defined in 
           <a target="_blank" href="https://github.com/giffordlabcvr/Deltaretrovirus-GLUE/blob/master/glue/erv/deltaErvSchemaExtensions.glue"><b>this file</b></a>
		   and comprise two additional tables: 'locus_data' and 'refcon_data'.
		   Both tables are linked to the main 'sequence' table via the 'sequenceID' field.
  		</p>


 		<p>
		   The '<b>locus_data</b>' table contains information pertaining to individual
		   ERV sequences: e.g. species in which they occur, genome assembly version, 
		   genomic location(i.e. scaffold, location coordinates, and orientation).
		</p>

 		<p>
		   The '<b>refcon_data</b>' table contains information pertaining to our
		   ERV reference sequences, which we have constructed in an effort to
		   reconstruct, as closely as possible, the sequences of the progenitor viruses
		   that gave rise to ERVs. 
		</p>




	   <br>
	   <h3>
		 <a id="Publications" class="anchor" href="#Publications" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Related Publications</strong>
	   </h3>
	   <hr>


		<p>
		
	   
		 Singer JB, Thomson EC, McLauchlan J, Hughes J, and RJ Gifford
		 <b>(2018)</b>
		 <br>
		 GLUE: A flexible software system for virus sequence data.
		 <br>
		 <i>BMC Bioinformatics</i>
		 <a href="https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-018-2459-9" target="_blank">[view]</a> 
		 <br>
		 <br>

		 Zhu H, Dennis T, Hughes J, and RJ Gifford
		 <b>(2018)</b>
		 <br>
		 Database-integrated genome screening (DIGS): exploring genomes heuristically using sequence similarity search tools and a relational database.
		 <a href="https://www.biorxiv.org/content/early/2018/04/25/246835" target="_blank">[preprint]</a> 
		 <br>
		 <br>

		 Hron T, Elleder D, and RJ Gifford
		 <b>(2019)</b>
		 <br>
		 Deltaretroviruses have circulated since at least the Paleogene and infected a broad range of mammalian species.
		 <br>
		 <i>Retrovirology</i>
		 <a href="https://retrovirology.biomedcentral.com/articles/10.1186/s12977-019-0495-9" target="_blank">[view]</a> 
		 <br>
		 <br>


		 Hron T, Farkašová H, Gifford RJ, Benda P, Hulva P, Tamás Görföl, Pačes J, and D. Elleder
		 <b>(2018)</b>
		 <br>
		 Remnants of an ancient deltaretrovirus in the genomes of horshoe bats (<i>Rhinolophidae</i>).
		 <i>Virus Research</i>
		 <a href="http://www.mdpi.com/1999-4915/10/4/185" target="_blank">[view]</a>
		 <br>
		 <br>

		 Gifford RJ, Blomberg B, Coffin JM, Fan H, Heidmann T, Mayer J, Stoye J, Tristem M, and WE Johnson
		 <b>(2018)</b>
		 <br>
		 Nomenclature for endogenous retrovirus (ERV) loci.
		 <br>
		 <i>Retrovirology</i>
		 <a href="https://retrovirology.biomedcentral.com/articles/10.1186/s12977-018-0442-1" target="_blank">[view]</a> 
		 <br>
		 <br>


		 Farkašová H, Hron T, Pačes J, Hulva P, Benda P, Gifford RJ, Elleder D.
		 <b>(2017)</b>     
		 <br>
		 Discovery of an endogenous Deltaretrovirus in the genome of long-fingered bats (Chiroptera: Miniopteridae).
		 <br>
		 <i>Proc Natl Acad Sci U S A.</i>
		 <a href="http://www.pnas.org/content/114/12/3145.full" target="_blank">[view]</a>
		 <br>
		 <br>

 		</p>


	 <footer class="site-footer">
	   <span class="site-footer-owner"><a href="https://github.com/giffordlabcvr/Deltaretrovirus-GLUE/">Deltaretrovirus-GLUE</a> is maintained by <a href="https://github.com/giffordlabcvr">giffordlabcvr</a>.</span>

	   <span class="site-footer-credits">This page was generated by <a href="https://pages.github.com">GitHub Pages</a> using the <a href="https://github.com/jasonlong/cayman-theme">Cayman theme</a> by <a href="https://twitter.com/jasonlong">Jason Long</a>.</span>
	 </footer>

    </section>


  </body>
  
</html>



