<!DOCTYPE html>
<html lang="en-us">

  <head>
  

    <meta charset="UTF-8">
    <title>Deltaretrovirus-GLUE by giffordlabcvr: Virus data</title>
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <link rel="stylesheet" type="text/css" href="stylesheets/normalize.css" media="screen">
    <link href='https://fonts.googleapis.com/css?family=Open+Sans:400,700' rel='stylesheet' type='text/css'>
    <link rel="stylesheet" type="text/css" href="stylesheets/stylesheet.css" media="screen">
    <link rel="stylesheet" type="text/css" href="stylesheets/github-light.css" media="screen">

  </head>

  <body>
    <section class="page-header">
    
      <h1 class="project-name">Deltaretrovirus-GLUE</h1>
      <h2 class="project-tagline">Resources for comparative genomic analysis of deltaretroviruses.</h2>

      <a href="../index.html" class="btn">Home</a>
      <a href="./paleoviruses.html" class="btn">Paleoviruses</a>
      <a href="https://github.com/giffordlabcvr/Deltaretrovirus-GLUE/zipball/master" class="btn">Download</a>
      <a href="https://github.com/giffordlabcvr/Deltaretrovirus-GLUE/tree/master" class="btn">View on GitHub</a>
        
    </section>

    <section class="main-content">

	   <h3>
		 <a id="GlueProject" class="anchor" href="#GlueProject" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Comparative genomic analysis of deltaretroviruses using Deltaretrovirus-GLUE</strong>
	   </h3>
	   <hr>
	   
	   
	   <p>
         <b>Deltaretrovirus-GLUE</b> is a data resource designed to facilitate comparative analysis of
         deltaretrovirus genomes. It was created using the <a href="http://glue-tools.cvr.gla.ac.uk/"><b>GLUE</b></a>
         software environment. 
	   </p>
   
	  
	   <p><img src="./images/deltaretrovirus-header2.jpg" alt="Deltaretrovirus host species" /></p>




<blockquote>
	    Some of the mammalian species known to harbour deltaretrovirus infections. Left to right: 
	    (i) guenons;
		(ii) geladas;
		(iii) orang utan;
		(iv) cattle.	 
</blockquote>


	   <br>

	   <p>

         The Deltaretrovirus-GLUE project not only contains data items required for
         comparative genomic analysis
         - i.e. virus genome sequences, multiple sequence alignments, genome feature annotations, and other
         sequence-associated data -
         but also, loading the project into the GLUE software framework
         creates a relational database that represents the semantic relationships
         between these data items. This creates powerful framework for performing
         <i>standardised</i> comparative analysis, via GLUE's command layer.
	   </p>



	   <br>
	   <h3>
		 <a id="virusReferenceSequences" class="anchor" href="#sequenceData" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Deltaretrovirus reference sequences</strong>
	   </h3>
	   <hr>
		
		 <p>

		   Deltaretrovirus-GLUE contains GLUE 
		   <a target="_blank" href="https://github.com/giffordlabcvr/Deltaretrovirus-GLUE/blob/master/glue/core/deltaMasterReferences.glue"><b>reference sequences</b></a>
		   for all known deltaretrovirus species.
		   Reference sequences are linked to auxiliary data in		 
		   <a target="_blank" href="https://github.com/giffordlabcvr/Deltaretrovirus-GLUE/blob/master/tabular/core/deltaretrovirus-reference-data.tsv"><b>tabular format</b></a>.

		 </p>
		
		 <p>

		   For each deltaretrovirus species, we have created a 'master' reference sequence, as follows:
		   <ul>
			 <li><b>Primate T-lymphotrophic virus type 1</b>: (<a target="_blank" href="https://www.ncbi.nlm.nih.gov/nuccore/J02029"><b>J02029</b></a>)</li>
			 <li><b>Primate T-lymphotrophic virus type 2</b>: (<a target="_blank" href="https://www.ncbi.nlm.nih.gov/nuccore/M10060"><b>M10060</b></a>)</li>
			 <li><b>Primate T-lymphotrophic virus type 3</b>: (<a target="_blank" href="https://www.ncbi.nlm.nih.gov/nuccore/DQ093792"><b>DQ093792</b></a>)</li>
			 <li><b>Primate T-lymphotrophic virus type 4</b>: (<a target="_blank" href="https://www.ncbi.nlm.nih.gov/nuccore/EF488483"><b>EF488483</b></a>)</li>
			 <li><b>Bovine leukemia virus </b>: (<a target="_blank" href="https://www.ncbi.nlm.nih.gov/nuccore/K02120"><b>K02120</b></a>)</li>
		   </ul>

		 </p>
		



	   <br>
	   <h3>
		 <a id="genomeFeatures" class="anchor" href="#sequenceData" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Deltaretrovirus genome features</strong>
	   </h3>
	   <hr>

		<p>
          Deltaretroviruses have enveloped, spherical virions and a small, circular DNA
          genome ~3 kilobases (Kb) in length. The genome is characterised by a highly
          streamlined organization incorporating extensive gene overlap - the open reading
          frame (ORF) encoding the viral polymerase (P) protein occupies most of the genome
          and typically overlaps at least one of the ORFs encoding the core (C), and
          surface (S) proteins. 
 		</p>
		
		<p>
         We defined a
          <a target="_blank" href="https://github.com/giffordlabcvr/Deltaretrovirus-GLUE/blob/master/glue/core/deltaFeatures.glue"><b>standard set of genome features</b></a>
          for deltaretroviruses and mapped the locations of these genome features
		  onto out chosen reference sequences
		  (<b>see <a target="_blank" href="https://github.com/giffordlabcvr/Deltaretrovirus-GLUE/blob/master/glue/core/deltaMasterReferences.glue">here</b></a>).		  
		 </p>
		




	   <br>
	   <h3>
		 <a id="Phylogenetic trees" class="anchor" href="#PhylogeneticTrees" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Phylogenetic trees</strong>
	   </h3>
	   <hr>


		<p>

          We used GLUE to implement an automated process for deriving midpoint rooted,
          annotated trees from the alignments included in our project.
 		</p>


		<p>
          Trees were constructed at distinct taxonomic levels:
        
		  <ol>
			<li> <a href="https://github.com/giffordlabcvr/Deltaretrovirus-GLUE/tree/master/trees/main-root"><strong>Genus-level</strong></a></li>
			<li> <a href="https://github.com/giffordlabcvr/Deltaretrovirus-GLUE/tree/master/trees/main-species"><strong>Species-level</strong></a></li>
		  </ol>

 		</p>


		<p>

          Species level trees were reconstructed from a filtered set representing
          <b>all available GenBank sequences</b> for that species, and for 
          distinct genes and ORFs in the deltaretrovirus genome.
          Taxa sets were filtered based on alignment coverage (minimum 50% coverage for the gene 
          feature under examination).

 		</p>


		<p>
          All trees are associated with <b>annotation files</b>, so that sequence-related information
          can be displayed on tree tips. 
          We recommend using <a target="_blank" href="https://github.com/rambaut/figtree"><b>FigTree</b></a>
          to view these trees.
 		</p>

		<p>
 		
          Note: some deltaretrovirus species have an insufficient number of published sequences
          for phylogeny reconstruction, once filters have been applied.
 		</p>
 		

	   <br>
	   <h3>
		 <a id="schemaExtensions" class="anchor" href="#sequenceData" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Project-specific schema extensions</strong>
	   </h3>
	   <hr>


		 <p>
		   Deltaretrovirus-GLUE extends GLUE's 
		   <a target="_blank" href="http://glue-tools.cvr.gla.ac.uk/#/coreSchema"><b>core schema</b></a>
		   through the incorporation of a number of additional fields in the 
		   sequence table, and a project-specific custom table: 'isolate'.
		   These schema extensions are defined <a target="_blank" href="https://github.com/giffordlabcvr/Deltaretrovirus-GLUE/blob/master/glue/core/deltaSchemaExtensions.glue"><b>here</b></a>.
		   The isolate table is linked to the main 'sequence' table via the sequence ID field.
		   It contains information pertaining to viral isolates,  e.g. species sampled, date and location of sample.
		 </p>






	   <br>
	   <h3>
		 <a id="Publications" class="anchor" href="#Publications" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Related Publications</strong>
	   </h3>
	   <hr>


		<p>
	   
		 Singer JB, Thomson EC, McLauchlan J, Hughes J, and RJ Gifford
		 <b>(2018)</b>
		 <br>
		 GLUE: A flexible software system for virus sequence data.
		 <br>
		 <i>BMC Bioinformatics</i>
		 <a href="https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-018-2459-9" target="_blank">[view]</a> 
		 <br>
		 <br>

		 Zhu H, Dennis T, Hughes J, and RJ Gifford
		 <b>(2018)</b>
		 <br>
		 Database-integrated genome screening (DIGS): exploring genomes heuristically using sequence similarity search tools and a relational database.
		 <a href="https://www.biorxiv.org/content/early/2018/04/25/246835" target="_blank">[preprint]</a> 
		 <br>
		 <br>



		</p>
		




	   <br>
	   <h3>
		 <a id="license" class="anchor" href="#license" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>License</strong>
	   </h3>
       <hr>

		 <p>
		   This project is licensed under the <a href="https://www.gnu.org/licenses/agpl-3.0.en.html">GNU Affero General Public License v. 3.0</a>.
		 </p>

	   <br>




	 <footer class="site-footer">
	   <span class="site-footer-owner"><a href="https://github.com/giffordlabcvr/Deltaretrovirus-GLUE/">Deltaretrovirus-GLUE</a> is maintained by <a href="https://github.com/giffordlabcvr">giffordlabcvr</a>.</span>

	   <span class="site-footer-credits">This page was generated by <a href="https://pages.github.com">GitHub Pages</a> using the <a href="https://github.com/jasonlong/cayman-theme">Cayman theme</a> by <a href="https://twitter.com/jasonlong">Jason Long</a>.</span>
	 </footer>

    </section>


  </body>
  
</html>



